File: src/main/java/Hello.java
import com.example.demo.Person;

public class Hello {
    public static void main(String[] args) {
        Person p = new Person("Tom",66.5f, 1.7f);
        p.hello();
        System.out.println(p.bmi());

        int score = 88;
        System.out.println(score > 80 && score < 90);
        System.out.println(score < 80 || score > 90);

        char c = 'A';
        System.out.println( c > 60);
        System.out.println( c < 60);
    }
}

File: src/main/kotlin/Hello.kt
fun main() {

    var s : String ? = "asdfghjkl"
//    s = null;
    println(s!!.length)
    println(s?.substring(3))
    println(s?.get(3))

//    Human().hello();
//    val h = Human("Jack", weight = 50f, height = 1.6f)
//    println(h.bmi())

//    val score = 88
//    println(score > 60)
//
//    val c = 'A'
//    println(c.toInt() > 60)

}

class Human(var name : String = "Jack", var weight: Float = 50f, var height: Float = 1.6f) {
    fun hello(){
        println("Hello Kotlin");
    }
    fun bmi(): Float{
        val bmi = weight / (height * height)
        return bmi;
    }

    init {
        println("test + $weight");
    }
}